software/cloud-dashboard/node-server/config/device-map.js

```diff
@@ -6,7 +6,7 @@
 module.exports={
 	locMap : {
 	"4b0031000d51353432393339": "ADPLKenyaC6304",
-        "300057000951343334363138": "ADPLKenyaN3763",
+        "3e001d000e51353432393339": "ADPLKenyaN6833",
```

Comment:
Ok, let's just keep the name "ADPLKenyaN3763" in order to get the data. We can change later.